pf/tests/provider_configure_test.go

```diff
@@ -20,71 +20,110 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/diag"
 	"github.com/hashicorp/terraform-plugin-framework/provider/schema"
 	"github.com/pulumi/providertest/replay"
+	"github.com/pulumi/pulumi-terraform-bridge/pf/tests/internal/cross-tests"
 	"github.com/pulumi/pulumi-terraform-bridge/pf/tests/internal/testprovider"
 	tfpf "github.com/pulumi/pulumi-terraform-bridge/pf/tfbridge"
 	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/resource"
 	"github.com/stretchr/testify/require"
+	"github.com/zclconf/go-cty/cty"
 )
 
 func TestConfigure(t *testing.T) {
-	t.Run("configiure communicates to create", func(t *testing.T) {
-		// Test interaction of Configure and Create.
-		//
-		// TestConfigRes will read stringConfigProp information the provider receives via Configure.
-		server, err := newProviderServer(t, testprovider.SyntheticTestBridgeProvider())
-		require.NoError(t, err)
-		testCase := `
-		[
-		  {
-		    "method": "/pulumirpc.ResourceProvider/Configure",
-		    "request": {
-		      "args": {
-			"stringConfigProp": "example"
-		      }
-		    },
-		    "response": {
-		      "supportsPreview": true,
-		      "acceptResources": true
-		    }
-		  },
-		  {
-		    "method": "/pulumirpc.ResourceProvider/Create",
-		    "request": {
-		      "urn": "urn:pulumi:test-stack::basicprogram::testbridge:index/testres:TestConfigRes::r1",
-		      "preview": false
-		    },
-		    "response": {
-		      "id": "id-1",
-		      "properties": {
-			"configCopy": "example",
-			"id": "id-1"
-		      }
-		    }
-		  }
-		]`
-		replay.ReplaySequence(t, server, testCase)
-	})
+	t.Run("string", crosstests.MakeConfigure(
+		schema.Schema{Attributes: map[string]schema.Attribute{
+			"k": schema.StringAttribute{Optional: true},
+		}},
+		map[string]cty.Value{"k": cty.StringVal("foo")},
+		resource.PropertyMap{"k": resource.NewProperty("foo")},
+	))
 
-	t.Run("booleans", func(t *testing.T) {
-		// Non-string properties caused trouble at some point, test booleans.
-		server, err := newProviderServer(t, testprovider.SyntheticTestBridgeProvider())
-		require.NoError(t, err)
+	t.Run("secret-string", crosstests.MakeConfigure(
+		schema.Schema{Attributes: map[string]schema.Attribute{
+			"k": schema.StringAttribute{Optional: true},
+		}},
+		map[string]cty.Value{"k": cty.StringVal("foo")},
+		resource.PropertyMap{"k": resource.MakeSecret(resource.NewProperty("foo"))},
+	))
```

Comment:
> This is the framework bits right, not the coverage yet? If it adds coverage can you comment in-line to facilitate review? Appreciated!

There is some coverage here, but it is not extensive. We have coverage for:

1. A plain string property
2. A secret string property
3. A plain bool property
4. A string (`"false"`) passed to a bool property.

A follow-up PR will add actual coverage.